cmd/blocks/cmd.go

```diff
@@ -0,0 +1,68 @@
+package blocks
+
+import (
+	"errors"
+	"fmt"
+
+	errs "github.com/onflow/flow-evm-gateway/models/errors"
+	"github.com/onflow/flow-evm-gateway/storage/pebble"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	flowGo "github.com/onflow/flow-go/model/flow"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "check-blocks-integrity",
+	Short: "Checks the EVM blocks integrity",
+	RunE: func(*cobra.Command, []string) error {
+		if databaseDir == "" || chain == "" {
+			return fmt.Errorf("databaseDir, chain flags must be provided")
+		}
+
+		pebbleDB, err := pebble.OpenDB(databaseDir)
+		if err != nil {
+			return fmt.Errorf("failed to open pebble db: %w", err)
+		}
+		defer pebbleDB.Close()
+		store := pebble.New(pebbleDB, log.Logger)
+
+		chainID := flowGo.ChainID(chain)
+		blocks := pebble.NewBlocks(store, chainID)
+
+		latestHeight, err := blocks.LatestEVMHeight()
+		if err != nil {
+			return fmt.Errorf("failed to get latest EVM height: %w", err)
+		}
+		log.Info().Msgf("Checking for missing EVM blocks up to EVM height %d", latestHeight)
+
+		var missingBlocks []uint64
+		for height := uint64(0); height <= latestHeight+2; height++ {
```

Comment:
why +2? maybe add some comments.

Reply:
Oh, daam, this was a leftover, to trigger missing blocks locally :sweat_smile:

Reply:
Nice catch :100:

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/775/files#diff-d84b766d58fea0435e590503d796ebee2a1377275ebdbb22b583b596944cb20aR40